src/transaction/log_recovery_redo_parallel.cpp

```diff
@@ -713,32 +713,35 @@ namespace cublog
   void
   redo_parallel::log_perf_stats () const
   {
-    const cubperf::statset_definition definition
-    {
-      perf_stats_async_definition_init_list
-    };
+    if ( perf_stats_is_active_for_async () )
```

Comment:
it is easier to do an early out if `!perf_stats_is_active_for_async ()`

Reply:
Done